static/app/api.tsx

```diff
@@ -502,6 +520,7 @@ export class Client {
           responseJSON,
           responseText,
           getResponseHeader: (header: string) => response.headers.get(header),
+          responseBlob,
```

Comment:
IMO, we should just pass `responseClone`, it'll be more flexible that way.

Reply:
Updated.